bin/i18n/translation_monitor/config_helper.rb

```diff
@@ -16,6 +16,8 @@ module ConfigHelper
   # explicitly specified. See the +merge_default_params+ function in this module.
   #   start_date: e.g. 2022-01-01 (yyyy-mm-dd). See https://developer.crowdin.com/croql/
   #   end_date: e.g. 2022-06-23
+  #   write_to_file: if true, the tool will write outputs to the local file system
```

Comment:
Does this mean the log messages or does it include the history file as well?

Reply:
Just the intermediate and final results: translations, source strings, and translation errors. I'll make the comment clearer.